WebContent/index.html

```diff
@@ -37,603 +37,606 @@ <h1>.svg</h1>
 		</div>
 	</div>
 
-	<div class="flexCol textFormat">
```

Comment:
unfortunately, it is just a single <div class="periodicTable"> outside your column divs, that appears as multiple changes in this diff here. In reality only two lines of code where changed here